dev-util/glslang/files/glslang-1.3.236-Install-static-libs.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/OGLCompilersDLL/CMakeLists.txt b/OGLCompilersDLL/CMakeLists.txt
```

Comment:
This patch and the 9999 one are idential, we should have a single version. glslang-Install-static-libs.patch and using ${PN} in both ebuilds should work

Reply:
Well, 1.3.231 and 1.3.236 DO differ (while 9999 and 236 don't)

Reply:
Sure, we can make a 1.3.231 and 1.3.236/9999 version then

Reply:
will `${PN}-1.3.236-Install-static-libs.patch` do?